components/todo-list.js

```diff
@@ -1,19 +1,19 @@
-import React from "react";
-import styled from "styled-components";
+import React, { useState } from 'react';
 
 import TodoItem from "./todo-item";
 
-export default function TodoList() {
+import styled from "styled-components";
+
+export default function TodoList({todos, todoRemove}) {
+
   return (
     <Wrapper>
-      Todo list가 들어가는 자리입니다!
-      <TodoItem />
+        {todos.map(todo => <TodoItem key={todo.id} {...todo} todoRemove={todoRemove}/>)}
```

Comment:
```suggestion
        {todos.map((todo, index) => <TodoItem key={JSON.stringify(todo)} {...{todo}} onDelete={onDelete(index)} />)}
```
1. key에 id를 써주셨는데 id가 진짜 id가 아니라 index이기도 했고, id 프로퍼티가 필요하지 않아 key를 다른 것으로 바꿔주시면 좋을 것 같아요.
이번 미션 같은 경우에 todo에 content와 date밖에 없기 때문에 겹칠 우려가 있는데, 이럴 때에는 객체를 string으로 바꿔서 key에 넣어주기도 한답니다. [참고](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify)
2. props의 이름과 넘겨줄 값을 담은 변수의 이름이 같다면 spread operator를 이용해 넘겨주시면 좋아요.
3. 저번 미션의 경우 profile 객체만 넘겨줬기 때문에 props에 값을 spread operator로 풀어줘서 넘겨주면 좋았는데요,
지금 같은 경우 todo를 풀어서 넘기면 content, date와 onDelete가 한 객체로 묶이는데,
그것보단 todo, onDelete가 같은 객체로 묶이는 게 더 어울릴 것 같아요!
위에처럼 넘겨주시면 됩니다~
4.
```pages/index.js````에서 handleDelete를 다음과 같이 정의했었죠!
```
  const handleDelete = (index) = () => {
    setTodos([ ...todo.slice(0, index), ...todo.slice(index+1, todo.length)]);
  }
```
TodoItem에게 ```onDelete``` props를 ```onDelete={onDelete(index)}``` 이렇게 넘겨주면
TodoList가 index까지 포함해서 보내기 때문에
TodoItem이 props로 onDelete를 받을 때 아래 함수에 자신의 index값이 채워진 채로 onDelete를 받게 됩니다.
```
() => {
    setTodos([ ...todo.slice(0, index), ...todo.slice(index+1, todo.length)]);
  }
```

Reply:
4번의 handleDelete가 아니라 todoRemove로 정의했었는데 이부분 말씀하시는거 맞나요?.?

Reply:
넹 그거 이름 바꾸면 좋을 것 같다고 리뷰 달았어요! index.js 파일부터 리뷰 확인해주시면 편할 거예요!